html5/js/lib/rencode.js

```diff
@@ -271,15 +263,6 @@ function rencode(obj) {
 			return rencode_dict(obj);
 		}
 		if(obj.constructor===Uint8Array) {
-			if (rencode_legacy_mode) {
-				//legacy rencode cannot handle bytearrays
-				const CHUNK_SZ = 0x8000;
-				const c = [];
-				for (let i=0; i < u8a.length; i+=CHUNK_SZ) {
-					c.push(String.fromCharCode.apply(null, u8a.subarray(i, i+CHUNK_SZ)));
-				}
-				return rencode_string(c.join(""));
-			}
```

Comment:
This removes legacy support - it does not belong in a bug fix.

Reply:
Oh, ok. I got the impression that support for legacy mode had already been dropped. I'll update the PR to only contain the bug fix.

Reply:
Legacy mode has been dropped from xpra v6, not from v5 due to be released.
The fix should be released for a version that can be used with older servers.
Then we can drop legacy mode.

Reply:
Understood, in that case I think reverting #259 is the best course of action.

Reply:
Meh. Now I am thinking that we should just merge this and drop all <v5 compatibility.
Those that still need to support older versions can use an older version of the html5 client.